### ISSUE TITLE
docs started from che 7 factory have broken links to "Che 7 Beta" doc section

### ISSUE THREAD

**Issue by ibuziuk** [eclipse/che-docs#728]
How to reproduce:
- start a workspace from https://github.com/eclipse/che-docs#building-online
[![Contribute](https://che.openshift.io/factory/resources/factory-contribute.svg)](https://che.openshift.io/f?id=factorycgpdptc6cjetbhr5)
- navigate to "che 7 Beta documentation"
- ERROR: page redirects to `http://0.0.0.0:4000/che-7/introduction-to-eclipse-che.html` which is not accessible 
- NOTE: link to che 6 docs works just fine
![Peek 2019-06-12 10-58](https://user-images.githubusercontent.com/1461122/59337736-72adfe80-8d01-11e9-8401-95e133b113cf.gif)

**Comment by ibuziuk** [comment]
cc: @rkratky @benoitf 
I have fixed plugin / editor format in the factory definition, but faced this issue afterwards. (not sure if it is che specific though)

**Comment by ibuziuk** [comment]
looks like che specific - locally seems to work just fine

**Comment by rkratky** [comment]
@ibuziukcould you please give me more details on how to reproduce this? I don't understand how you get to the doc page from the workspace (other than clicking the 'Docs' button on the Dashboard, but that redirects to https://www.eclipse.org/che/docs/, from which the link to Che 7 docs works fine).

**Comment by ibuziuk** [comment]
there is a dedicated factory in the repos readme:

[![Contribute](https://che.openshift.io/factory/resources/factory-contribute.svg)](https://che.openshift.io/f?id=factorycgpdptc6cjetbhr5)

**Comment by slemeur** [comment]
please check @sunix : https://github.com/sunix/devfiles/blob/master/che-sites/docs-sites.devfile.yaml

**Comment by dneary** [comment]
When I run the task "che:start" I see a "_start" tab containing:

```
Configuration` file: /projects/che-docs/src/main/_config.yml
            Source: /projects/che-docs/src/main
       Destination: /projects/che-docs/src/main/_site
 Incremental build: enabled
      Generating... 
   GitHub Metadata: No GitHub API authentication could be found. Some fields may be missing or have incorrect data.
                    done in 16.17 seconds.
 Auto-regeneration: enabled for '/projects/che-docs/src/main'
    Server address: http://0.0.0.0:4000//
  Server running... press ctrl-c to stop.
```

In the "Previews" tab, I can click on "Go to" and see a preview of the live docs welcome page, but when I click on the "Che 7 Beta" link, it brings me to the 0.0.0.0:4000 link Ilya referred to. When I click on "Preview" instead, it opens a preview pane, which does not seem to work either.

**Comment by rkratky** [comment]
Fixed in #740.
